recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,4 @@
+
+
+
+if errorlevel 1 exit 1
```

Comment:
Rather than try to iterate on that in this PR, you are welcome to do it as a PR on the feedstock. In the meantime I recommend skipping the windows build through the staged-recipes phase.